L1Trigger/TrackFindingTMTT/src/HTrphi.cc

```diff
@@ -0,0 +1,685 @@
+#include "L1Trigger/TrackFindingTMTT/interface/HTrphi.h"
+#include "L1Trigger/TrackFindingTMTT/interface/InputData.h"
+#include "L1Trigger/TrackFindingTMTT/interface/TP.h"
+#include "L1Trigger/TrackFindingTMTT/interface/L1fittedTrack.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Settings.h"
+#include "L1Trigger/TrackFindingTMTT/interface/PrintL1trk.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include <vector>
+#include <limits>
+#include <atomic>
+#include <sstream>
+#include <mutex>
+
+using namespace std;
+
+namespace tmtt {
+
+  //=== The r-phi Hough Transform array for a single (eta,phi) sector.
+  //===
+  //=== Its axes are (q/Pt, phiTrk), where phiTrk is the phi at which the track crosses a
+  //=== user-configurable radius from the beam-line.
+
+  namespace {
```

Comment:
Unlike the previous case, having this in a a anonymous namespace in a .cc file is not a problem.

I would still suggest moving this to a `static` variable inside the `HTrphi::HTrphi` constructor to keep it more "local" to where it is used.

Reply:
done